db/src/schema/deployment.rs

```diff
@@ -91,6 +91,18 @@ pub struct Deployment {
     pub values_override: Option<serde_json::Value>,
 }
 
+#[derive(QueryableByName)]
+pub struct DeploymentStat {
+    #[sql_type = "diesel::sql_types::BigInt"]
+    pub count: i64,
+    #[sql_type = "diesel::sql_types::Varchar"]
+    pub kind: String,
+    #[sql_type = "diesel::sql_types::Varchar"]
+    pub status: String,
```

Comment:
Would `DeploymentStatus` work here?
```suggestion
    pub status: DeploymentStatus,
```